Mathlib/Topology/ContinuousFunction/Compact.lean

```diff
@@ -26,12 +26,7 @@ you should restate it here. You can also use
 
 noncomputable section
 
-open scoped Classical
-open Topology NNReal BoundedContinuousFunction
-
-open Set Filter Metric
-
-open BoundedContinuousFunction
+open Topology NNReal BoundedContinuousFunction Set Filter Metric
```

Comment:
I think you're mangling what was originally `open_locale` with the bare `open`. Can you restore what was there before, but replacing one of the `open` with `open scoped`

Reply:
I am aware of this issue in general. In this case, replacing the `open` by `open scoped` errors.